extensions/WEBGL_clip_cull_distance/extension.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<draft href="WEBGL_clip_cull_distance/">
+  <name>WEBGL_clip_cull_distance</name>
+
+  <contact> <a href="https://www.khronos.org/webgl/public-mailing-list/">WebGL
+  working group</a> (public_webgl 'at' khronos.org) </contact>
+
+  <contributors>
+    <contributor>Members of the WebGL working group</contributor>
+    <contributor>Kevin Rogovin (kevinrogovin 'at' invisionapp.com)</contributor>
+  </contributors>
+
+  <number>43</number>
+
+  <depends>
+    <api version="2.0"/>
+  </depends>
+
+  <overview>
+    <mirrors href="https://www.khronos.org/registry/OpenGL/extensions/EXT/EXT_clip_cull_distance.txt"
```

Comment:
Should this mirror https://chromium.googlesource.com/angle/angle/+/HEAD/extensions/ANGLE_clip_cull_distance.txt now? Since the extension text below mentions `ANGLE_clip_cull_distance`, that seems more appropriate.

Reply:
The ANGLE extension is not self-contained and plain text files do not have hyperlinks, so it seems to me that linking directly to the original EXT extension is more useful to readers. All WebGL-specific differences are mentioned right here anyway.

Reply:
Would you link to https://chromium.googlesource.com/angle/angle/+/HEAD/extensions/ANGLE_clip_cull_distance.txt in a separate hyperlink in this extension spec instead?

Reply:
Done.

Reply:
Thanks.